builtin/providers/vsphere/resource_vsphere_file.go

```diff
@@ -0,0 +1,254 @@
+package vsphere
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/vmware/govmomi"
+	"github.com/vmware/govmomi/find"
+	"github.com/vmware/govmomi/object"
+	"github.com/vmware/govmomi/vim25/soap"
+	"golang.org/x/net/context"
+)
+
+type file struct {
+	datacenter      string
+	datastore       string
+	sourceFile      string
+	destinationFile string
+}
+
+func resourceVSphereFile() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceVSphereFileCreate,
+		Read:   resourceVSphereFileRead,
+		Update: resourceVSphereFileUpdate,
```

Comment:
got it - in that case you can exclude `ForceNew: false` from that parameter - that is the default :)